arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -805,6 +806,25 @@ when disk size is very constrained and no replication is used.)");
               arangodb::options::Flags::OnSingle))
       .setIntroducedIn(31005);
 
+  options
```

Comment:
This option should not be made available in regular releases. I suggest we restrict it to either maintainer mode or failure tests, e.g. `#ifdef ARANGODB_ENABLE_MAINTAINER_MODE` or `#ifdef ARANGODB_ENABLE_FAILURE_TESTS`.

Reply:
Done.